spec/requests/graphql/mutations/delete_eras_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Types::QueryType do
+  describe 'display eras' do
+    it 'can delete an era' do
+      user = create(:user, :with_eras_events)
+
+      post graphql_path, params: { query: query(id: user.eras[0].id) }
+      result = JSON.parse(response.body)
+
+      require "pry"; binding.pry
+      data = result['data']['deleteEra']
+    end
+  end
+
+  def query(era_id)
+    require "pry"; binding.pry
```

Comment:
Pry's have been left in for now because it's not in a working form yet.  I need some assistance in figuring out the next steps here, and would love to pair up!